util/src/mem.rs

```diff
@@ -150,11 +245,16 @@ impl<T: Default> SwapWithDefaultExt for T {
 }
 
 /// Extension trait that provides a method to explicitly move values.
+///
+/// # Examples
+///
+/// ```rust
+/// use rosenpass_util::mem::MoveExt;
+/// let val = 42.move_here();
```

Comment:
Is there a better example for this? To me it looks like `let val = 42; assert_eq!(val, 42);` return fine, too.

Reply:
I think in Rust you can't directly "test" if a value has been moved because once a value is moved, it becomes inaccessible. 

I can add an additional comment that explains that the initial variable will be inaccessible. 

Alternatively, I can test if the value is dropped by using a custom type that signals as soon as it is dropped (setting an AtomicBool for example). But I think that that would be an "overkill" example and I can instead add it as a unittest.

Reply:
Okay if you think this is common knowledge I'm fine with it.